lib/implore.es6.js

```diff
@@ -121,7 +121,16 @@ export default function implore(request, callback) {
 			response.body = xhr.responseText;
 			response.status = xhr.status;
 			response.type = xhr.getResponseHeader('Content-Type');
-			response.headers = xhr.getAllResponseHeaders();
+			const rawHeaders = xhr.getAllResponseHeaders();
+			// Just in case some headers don't get processed properly
```

Comment:
I don't understand what this comment means in the context of the two lines immediately surrounding it.